sdk/src/wallet/account/operations/participation/voting.rs

```diff
@@ -64,8 +64,6 @@ where
             }
         }
 
-        // TODO check if answers match the questions ?
```

Comment:
This wouldn't allow to participate without registering an event before, don't think it's worth it